phoenix-ios/phoenix-ios/views/widgets/AppStatusPopover.swift

```diff
@@ -0,0 +1,620 @@
+					.read(titleIconWidthReader)
+					.frame(width: titleIconWidth, alignment: .center)
+			}
+			.font(.title3)
+			.padding(.bottom, 15)
+			
+			PendingSettingsDetails(pendingSettings: value).font(.callout)
+		}
+	}
+	
+	@ViewBuilder
+	func syncStatusSection_syncState() -> some View {
+		
+		VStack(alignment: .leading) {
+			
+			switch syncState {
+				case .initializing:
+					
+					Label {
+						Text("Cloud sync initializing")
+					} icon: {
+						Image(systemName: "arrow.clockwise.icloud")
+							.imageScale(.medium)
+							.read(titleIconWidthReader)
+							.frame(width: titleIconWidth, alignment: .center)
+					}
+					.font(.title3)
+					
+				case .updatingCloud(let details):
+					
+					Label {
+						Text("Updating cloud")
+					} icon: {
+						Image(systemName: "icloud")
+							.imageScale(.medium)
+							.read(titleIconWidthReader)
+							.frame(width: titleIconWidth, alignment: .center)
+					}
+					.font(.title3)
+					.padding(.bottom, 10)
+					
+					switch details.kind {
+						case .creatingRecordZone:
+							Text("Creating record zone...").font(.callout)
+						case .deletingRecordZone:
+							Text("Deleting record zone...").font(.callout)
+					}
+					
+				case .downloading(let progress):
+					
+					Label {
+						Text("Downloading payments")
+					} icon: {
+						Image(systemName: "icloud.and.arrow.down")
+							.imageScale(.medium)
+							.read(titleIconWidthReader)
+							.frame(width: titleIconWidth, alignment: .center)
+					}
+					.font(.title3)
+					.padding(.bottom, 15)
+					
+					SyncProgressDetails(syncState: $syncState, syncProgress: progress)
+						.font(.callout)
+				
+				case .uploading(let progress):
+					
+					Label {
+						Text("Uploading payments")
+					} icon: {
+						Image(systemName: "icloud.and.arrow.up")
+							.imageScale(.medium)
+							.read(titleIconWidthReader)
+							.frame(width: titleIconWidth, alignment: .center)
+					}
+					.font(.title3)
+					.padding(.bottom, 15)
+					
+					SyncProgressDetails(syncState: $syncState, syncProgress: progress)
+						.font(.callout)
+				
+				case .waiting(let details):
+					
+					switch details.kind {
+						case .forInternet:
+							
+							Label {
+								Text("Cloud sync paused")
+							} icon: {
+								Image(systemName: "icloud.slash")
+									.imageScale(.medium)
+									.read(titleIconWidthReader)
+									.frame(width: titleIconWidth, alignment: .center)
+							}
+							.font(.title3)
+							.padding(.bottom, 15)
+							
+							Text("Waiting for internet")
+								.font(.callout)
+							
+						case .forCloudCredentials:
+							
+							Label {
+								Text("Cloud sync error")
+							} icon: {
+								Image(systemName: "exclamationmark.icloud")
+									.imageScale(.medium)
+									.read(titleIconWidthReader)
+									.frame(width: titleIconWidth, alignment: .center)
+							}
+							.font(.title3)
+							.padding(.bottom, 15)
+							
+							Text("Please sign into iCloud")
```

Comment:
nit: change this wording to `Sign into iCloud to backup your payments` to give some context.

Reply:
Done in c611412